.github/workflows/publish_openrpc_document.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish OpenRPC document
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build_release_binary_and_publish_document:
+    name: Build nimiq-rpc-schema with release flag
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install protoc
+        run: |
+          sudo apt-get update && sudo apt-get install -y protobuf-compiler
+      - uses: dtolnay/rust-toolchain@stable
+      - name: Build binary in release mode
+        run: |
+          cargo build --release --bin nimiq-rpc-schema
+      - name: Get latest release
```

Comment:
I agree, if there's a tag but no release, this will break, or even worse, upload it's generated RPC spec to an older release. Can the trigger for this action be changed to trigger on release, not on tag?

Reply:
I set the trigger to `on release created`